Tests/UIKit/UIDatePickerTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIDatePickerTests.swift
+//  Rex
+//
+//  Created by Guido Marucci Blas on 3/25/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+import XCTest
+import Rex
+
+class UIDatePickerTests: XCTestCase {
+    
+    var date: NSDate!
+    var picker: UIDatePicker!
+    
+    override func setUp() {
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "MM/dd/YYYY"
+        date = formatter.dateFromString("11/29/1988")!
+        
+        picker = UIDatePicker(frame: CGRectZero)
+    }
+    
+    func testUpdatePickerFromProperty() {
+        picker.rex_date.value = date
+        
+        XCTAssertEqual(picker.date, date)
+    }
+    
+    func testUpdatePropertyFromPicker() {
+        let expectation = self.expectationWithDescription("Expected rex_date to send an event when picker's date value is changed by a UI event")
+        defer { self.waitForExpectationsWithTimeout(2, handler: nil) }
```

Comment:
My guess is that this blocks the test "UI thread" preventing the action event from being delivered and the expectation from being fulfilled. I'm not at all familiar with the threading of XCTest so not sure what can be done to make this work